docs/user/user/howto/real-time-events/real-time-events.md

```diff
@@ -0,0 +1,37 @@
+# "Real-Time" Events
+
+## Defining "Real-Time" Glean Events
+
+"Real-Time" in the context of Glean [Events](../../../reference/metrics/event.md) directly relates to configuring Glean to send each event in an [Events Ping](../../pings/events.md) as soon as it is recorded.
+
+## Configuring Glean For "Real-Time" Events
+
+Glean event ping submission can either be configured at [initialization](../../../reference/general/initializing.md) or through [Server Knobs](../../../user/server-knobs/other/max-events.md).
+
+Setting the maximum event threshold to a value of `1` will configure Glean to submit an Events Ping for every event recorded.
+
+## Considerations
+
+### What "Real-Time" Glean Events Are _Not_
+
+Configuring Glean to submit events as soon as they are recorded does not mean to imply that the event data is available for analysis in
+real-time. There are networks to traverse, ingestion pipelines, etl, etc. that are all factors to keep in mind when considering how soon
+the data is available for analysis purposes. This documentation only purports to cover configuring Glean to send the data in a real-time
+fashion and does not make any assumptions about the analysis of data in real-time.
+
+### More Network Requests
+
+There are also other trade-offs to consider when sending event pings with each and every event. For every event recorded a network request
```

Comment:
Might just omit this first sentence. Not sure it adds anything the structure of the document doesn't itself communicate.